src/features/home/components/BedsMigrationModal.tsx

```
@@ -0,0 +1,158 @@
+          <div className="grid grid-cols-4 mb-2 w-full">
+            {beds.map((bed, i) => {
+              const equipment = [bumpkin, ...Object.values(farmHands)].map(
+                (f) => f.equipped,
+              )[i];
+              return (
+                <BedCell
+                  bed={bed}
+                  equipment={equipment}
+                  isPlaced
+                  key={bed}
+                />
```

ESLint

[error] Replace `⏎··················bed={bed}⏎··················equipment={equipment}⏎··················isPlaced⏎··················key={bed}⏎···············` with `·bed={bed}·equipment={equipment}·isPlaced·key={bed}`

(prettier/prettier)